button-card.js

```diff
@@ -107,8 +107,12 @@ class ButtonCard extends LitElement {
     }
     paper-button {
       display: flex;
+      flex-direction: column;
       margin: auto;
       text-align: center;
+      align-items: center;
+      text-transform: uppercase;
```

Comment:
I never had an instance displaying text. I just assumed the previous `paper-button` did it uppercase based on the screenshots in the `button-card` README